Zotero/Scenes/Detail/CopyBibliography/Views/CopyBibliographyViewController.swift

```diff
@@ -0,0 +1,171 @@
+//
+//  CopyBibliographyViewController.swift
+//  Zotero
+//
+//  Created by Miltiadis Vasilakis on 27/12/23.
+//  Copyright © 2023 Corporation for Digital Scholarship. All rights reserved.
+//
+
+import UIKit
+import WebKit
+
+import CocoaLumberjackSwift
+import RxSwift
+
+class CopyBibliographyViewController: UIViewController {
+    // MARK: - Types
+    private enum OverlayState {
+        case processing
+        case error(String)
+    }
+
+    // MARK: - Properties
+    private weak var overlayBody: UIView!
+    private weak var overlayActivityIndicator: UIActivityIndicatorView!
+    private weak var overlayErrorIcon: UIImageView!
+    private weak var overlayText: UILabel!
+    private weak var webView: WKWebView!
+
+    private let viewModel: ViewModel<CopyBibliographyActionHandler>
+    private let disposeBag = DisposeBag()
+
+    weak var coordinatorDelegate: DetailCopyBibliographyCoordinatorDelegate?
+
+    // MARK: - Object Lifecycle
+    init(viewModel: ViewModel<CopyBibliographyActionHandler>) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    deinit {
+        viewModel.process(action: .cleanup)
+        DDLogInfo("CopyBibliographyViewController deinitialized")
+    }
+
+    // MARK: - View Lifecycle
+    override func loadView() {
+        super.loadView()
+
+        let overlayBody = UIView()
```

Comment:
`super.loadView()` in this case is equal to `view = UIView()`, but you are right, better not call it, unless there is reason to. 
I have to say, I don't totally agree with the documented guideline. In cases where there is a mix of interface builder and programmatic UI, I think that `loadView` is a good place semantically to create all the UI you would have also build with the interface builder, including constraints. However, I'm happy to have us agree on a certain guideline and adhere to it in the codebase. Would that be, only create the main `view` of the controller in `loadView`, whenever this is needed for a reason, and setup the rest of the views in `viewDidLoad`?

Reply:
I agree that `loadView` would kind of make more sense, but I'd still rather adhere to their recommendation, whatever reason they may have (some time ago I read somewhere that `loadView` should return as fast as possible and you can do whatever needed in `viewDidLoad` 🤷‍♂️).

Also the whole codebase initializes views in `viewDidLoad` currently and it's such a small detail that I don't think we need to rethink this one.

So if you need to create some kind of special root view you can do that in `loadView` (otherwise if it's just a `UIView` then `loadView` doesn't even need to be implemented). Then create all views, adjustments and constraints in `viewDidLoad`.

Reply:
Check https://github.com/zotero/zotero-ios/pull/827/commits/a40cb59c457b715c97449228f79febb8bb01157e